Directory.Build.targets

```
@@ -100,18 +117,16 @@ or
     <RemoveDir Directories="$(SentryAndroidArtifactsDestination)" ContinueOnError="true" />
   </Target>
 
-  <Target Name="CleanNativeSDK" AfterTargets="Clean" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
+  <Target Name="CleanWindowsSDK" AfterTargets="Clean" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
     <RemoveDir Directories="$(SentryNativeRoot)build/" ContinueOnError="true" />
     <RemoveDir Directories="$(SentryNativeRoot)install/" ContinueOnError="true" />
     <RemoveDir Directories="$(SentryNativeWindowsArtifactsDestination)" ContinueOnError="true" />
   </Target>
 
-  <!-- Build the iOS SDK: dotnet msbuild /t:BuildCocoaSDK src/Sentry.Unity -->
-  <Target Name="BuildCocoaSDK"
-          Condition="$([MSBuild]::IsOSPlatform('OSX'))
+  <!-- Build the iOS SDK: dotnet msbuild /t:BuildiOSSDK src/Sentry.Unity -->
+  <Target Name="BuildiOSSDK" Condition="$([MSBuild]::IsOSPlatform('OSX'))
           And '$(MSBuildProjectName)' == 'Sentry.Unity'
-          And (!Exists('$(SentryCocoaDeviceArtifactsDestination)') OR !Exists('$(SentryCocoaSimulatorArtifactsDestination)'))"
-          BeforeTargets="BeforeBuild">
+          And (!Exists('$(SentryCocoaDeviceArtifactsDestination)') OR !Exists('$(SentryCocoaSimulatorArtifactsDestination)'))" BeforeTargets="BeforeBuild">
```

With the change `BuildCocoaSDK` -> `BuildiOSSDK` we might want to rename also:

```suggestion
          And !Exists('$(SentryiOSDeviceArtifactsDestination)/sentry.dll')" DependsOnTargets="BuildNativeSDK" BeforeTargets="BeforeBuild">
```

---

Replaced `CocoaDevice -> iOSDevice` and `CocoaSimulator->iOSSimulator` in the file